services/121-service/src/payments/dto/payment-return.dto.ts

```diff
@@ -38,4 +40,15 @@ export class PaymentReturnDto {
     type: CountAndTransferValueDto,
   })
   approved: CountAndTransferValueDto;
+
+  @ApiProperty({
+    type: () => PartialType(TransactionViewEntity),
+    isArray: true,
+  })
+  fsps: Pick<
+    TransactionViewEntity,
+    | 'programFspConfigurationName'
+    | 'programFspConfigurationId'
```

Comment:
we don't use id, so I wouldn't include it here for a bit more focus. If we need it later, we'll add it then.